src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -202,7 +208,7 @@ protected function _getPortableDatabaseDefinition(array $database): string
      */
     protected function _getPortableSequenceDefinition(array $sequence): Sequence
     {
-        if ($sequence['schemaname'] !== 'public') {
+        if ($sequence['schemaname'] !== $this->getCurrentSchema()) {
```

Comment:
This is technically also a bug fix but I don't think anybody uses sequences with Postgres at this point.